### ISSUE TITLE
Recommendation to fine-tune ALMA-X to parallel data

### ISSUE THREAD

**Issue by xonlp14** [fe1ixxu/ALMA#75]
Hi!

First of all, I wanted to give you kudos for the fantastic paper and piece of code you made :)
Secondly, I was wondering what would be the recommended strategy to finetune an alma-x model for a given group with 200K parallel segments in a language pair. 
I tried to fine tune by imitating your "Post-Training Stage 1: Supervised Fine-tuning" with lora as in parallel_ft_lora but I see how the model has a good translation quality at step 0 and just gets objectively worse after every step although the validation and training loss improve. 
- Is this is a good idea? If yes, what would be your recommended hyperparameters or important steps to consider?
- Should I preferably fine-tune it using Adaptive Rejection Preference Optimization (ARPO) ? If so, are you planning to release the code you used for such task?

Thanks again!

**Comment by WuXuanxin** [comment]
It would be great if the author could make the ARPO code available.
